src/controllers/controller.line.js

```diff
@@ -315,4 +315,19 @@ module.exports = DatasetController.extend({
 		model.borderWidth = valueOrDefault(options.hoverBorderWidth, options.borderWidth);
 		model.radius = valueOrDefault(options.hoverRadius, options.radius);
 	},
+
+	setDatasetHoverStyle: function() {
+		var line = this.getMeta().dataset;
+		var model = line._model;
+		var config = this._config;
+		var getHoverColor = helpers.getHoverColor;
```

Comment:
We usually define this at the top of the file, right? Then you don't need to define it both here and in `setHoverStyle`

Reply:
Oh, right :)

Reply:
Actually, `getHoverColor` is still in `core/core.helpers.js` and thus not available at top of file.
I'm not in the mood of writing unit tests for those right now, so not moving it :)